test/unit/client.js

```diff
@@ -580,6 +580,20 @@ describe('Client WSFallback', () => {
 });
 
 describe('Channel.queryChannels', async () => {
+	it('should not hydrate activeChannels and channel configs when disableCache is true', async () => {
```

Comment:
@isekovanic could you, please, add test also for channel.query()?

Reply:
Yup